demos/projects/ESPRESSIF/adu/README.md

```diff
@@ -35,6 +35,11 @@ This sample will allow you to update an ESP32 over the air (OTA) using Azure Dev
     For other Operating Systems or to update an existing installation, follow [Espressif official documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/index.html#get-started).
```

Comment:
Please do not merge until [this](https://github.com/Azure/azure-sdk-for-c/compare/adu/patch) is merged.

Reply:
https://github.com/Azure/azure-sdk-for-c/pull/2437

Link here. Also note this has to be pulled up to the middleware as well.